### ISSUE TITLE
Is there any commonality between RPPG and PPG signals?

### ISSUE THREAD

**Issue by jungyin** [ubicomplab/rPPG-Toolbox#310]
I hope to train my model to obtain a usable RPPG signal extraction model, but I want to validate its accuracy outside of the test and validation sets. Can the PGG signals collected using PGG devices be used for validation?

**Comment by yahskapar** [comment]
Hi @jungyin,

Can you elaborate a bit on this? Are you talking about collecting your own videos and comparing predicted signals to ground truth signals from your own PPG device?

If so, that should be doable. You'd just have to create a custom dataloader as described [here](https://github.com/ubicomplab/rPPG-Toolbox?tab=readme-ov-file#open_file_folder-adding-a-new-dataset) and then update your config file to use that as the test set. Let us know if you give that a try and run into any trouble.

**Comment by jungyin** [comment]
Thank you very much for your prompt reply. Yes, I would like to compare the collected signals with those collected by the PPG device, as well as calculate various physical signs through Heartpy. Currently, the respiratory rate I have calculated fluctuates significantly, and I need to compare it with my PPG signal to ensure that the current output content is more accurate

**Comment by jungyin** [comment]
Thank you very much for your prompt reply. Yes, I would like to compare the collected signals with those collected by the PPG device, as well as calculate various physical signs through Heartpy. Currently, the respiratory rate I have calculated fluctuates significantly, and I need to compare it with my PPG signal to ensure that the current output content is more accurate

**Comment by jungyin** [comment]
> Hi @jungyin,
> 
> Can you elaborate a bit on this? Are you talking about collecting your own videos and comparing predicted signals to ground truth signals from your own PPG device?
> 
> If so, that should be doable. You'd just have to create a custom dataloader as described [here](https://github.com/ubicomplab/rPPG-Toolbox?tab=readme-ov-file#open_file_folder-adding-a-new-dataset) and then update your config file to use that as the test set. Let us know if you give that a try and run into any trouble.

Thank you very much for your prompt reply. Yes, I would like to compare the collected signals with those collected by the PPG device, as well as calculate various physical signs through Heartpy. Currently, the respiratory rate I have calculated fluctuates significantly, and I need to compare it with my PPG signal to ensure that the current output content is more accurate

**Comment by yahskapar** [comment]
I'm assuming you figured out how to do this based on a combination of my previous reply and some of the follow-up issues you posted, so I will go ahead and close this issue.
